01_fib/NicholeL/fib.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func abs(args int) (res int) {
+	if args == 0 {
+		return 0
+	}
+	if args > 0 {
+		return args
+	}
+	return args * (-1)
+
+}
+
+func fib(args int) {
```

Comment:
use plural only for slices/arrays and maybe maps.
change to arg or val

Reply:
will update